src/PhoXiInterface.cpp

```diff
@@ -11,6 +11,14 @@ namespace phoxi_camera {
 
     }
 
+    // scanners IP obtaining methods
+    int countRowsWithStartingSign(char, std::vector<std::string>);
```

Comment:
Add some function description. Check descriptions in PhoXIinterface.h

Reply:
moved to PhoXIinterface.h, descriptions added